dask_sql/physical/rel/logical/limit.py

```diff
@@ -80,17 +79,17 @@ def _apply_offset(self, df: dd.DataFrame, offset: int, end: int) -> dd.DataFrame
         # its pandas representation at this point and we can calculate the cumsum
         # (which is not possible on the dask object). Recalculating it should not cost
         # us much, as we assume the number of partitions is rather small.
-        def select_from_to(df, partition_index, partition_borders):
-            partition_borders = partition_borders.cumsum().to_dict()
+        def select_from_to(index, partition, partition_borders):
+            partition_borders = partition_borders.compute().cumsum().to_dict()
```

Comment:
One difference between `from_map` and `from_delayed` is that Dask objects would already be computed by the time the function is called - is this an intended change?